src/components/svg-wrapper.tsx

```diff
@@ -267,6 +328,55 @@ const SvgWrapper = () => {
             onKeyDown={handleKeyDown}
         >
             <SvgCanvas />
+            {mode === 'select' && selectStart.x != 0 && selectStart.y != 0 && (
+                <g id="select_in_progress___no_use" transform={`translate(${calcSelectSX()}, ${calcSelectSY()})`}>
```

Comment:
`id` may not be necessary. Is it being used elsewhere?

Reply:
No use. I just copied it from somewhere.